src/tests/Microsoft.PowerFx.Interpreter.Tests/CustomFunctions.cs

```diff
@@ -68,6 +68,11 @@ public void CustomFunctionErrorOrBlank(string script, string expectedResult, boo
         // Must have "Function" suffix. 
         private class TestCustomFunction : ReflectionFunction
         {
+            public TestCustomFunction():
```

Comment:
Did you intend to add this?
This will cause Scan() to be skipped for this function and change test cases.

Reply:
you are right! this was by mistake...